crates/bevy_ecs/src/system/query.rs

```diff
@@ -477,7 +477,7 @@ impl<'w, 's, D: QueryData, F: QueryFilter> Query<'w, 's, D, F> {
     #[inline]
     pub fn iter(&self) -> QueryIter<'_, 's, D::ReadOnly, F> {
         // SAFETY:
-        // - `self.world` has permission to access the required components.
+        // - `self.world` has permission to access the related components.
```

Comment:
This doesn't appear to be related to required/related components, I think this is just a find&replace oopsie!